packages/models-library/src/models_library/utils/pydantic_tools_extension.py

```diff
@@ -20,8 +19,6 @@ def parse_obj_or_none(type_: type[T], obj) -> T | None:
 # Two usage styles:
 #
 # class Model(BaseModel):
-#     value: FieldNotRequired(description="some optional field")
 #     other: Field(NOT_REQUIRED, description="alternative")
 #
 NOT_REQUIRED: Final = None
```

Comment:
maybe this should also go away?

Reply:
no this is still used

Reply:
well removed it. was used in only one module